webapp/src/cloudsync.ts

```diff
@@ -120,8 +120,8 @@ export class ProviderBase {
             pxt.storage.setLocal(this.name + CLOUD_USER, JSON.stringify(user))
         else
             pxt.storage.removeLocal(this.name + CLOUD_USER);
-        data.invalidate("sync:user")
-        data.invalidate("github:user")
+            pxt.data.invalidate("sync:user")
```

Comment:
```suggestion
        pxt.data.invalidate("sync:user")
```

small: extra indent, these aren't meant to be part of the else above, right?